boards/adafruit_feather.json

```diff
@@ -0,0 +1,52 @@
+{
+  "build": {
+    "core": "earlephilhower",
+    "cpu": "cortex-m0plus",
+    "extra_flags": "-D ARDUINO_ADAFRUIT_FEATHER_RP2040 -DARDUINO_ARCH_RP2040 -DUSBD_MAX_POWER_MA=250",
+    "f_cpu": "133000000L",
+    "hwids": [
+      [
+        "0x2E8A",
+        "0x00C0"
+      ]
+    ],
+    "mcu": "rp2040",
+    "arduino": {
+      "earlephilhower": {
+        "variant": "adafruit_feather",
+        "boot2_source": "boot2_w25x10cl_4_padded_checksum.S",
+        "usb_vid": "0x239a",
+        "usb_pid": "0x80f1",
+        "usb_manufacturer": "Adafruit",
+        "usb_product": "Feather RP2040"
```

Comment:
Maybe we can get rid of these option by dynamically generating them from the `vendor` and `name` sections below?

Reply:
Indeed the builder script would actually do that [here](https://github.com/earlephilhower/arduino-pico/blob/3bcbb24603aa3ece5a665deeb12a6adfee341931/tools/platformio-build.py#L140-L151), this comes from boards supporting both cores where e.g. for the Pico, that would differ. But for boards only supporting one core, the product and manufacturer name is indeed always equal to the name and vendor field [here](https://github.com/earlephilhower/arduino-pico/blob/master/tools/makeboards.py#L177-L233), so that can be simplified.

Reply:
Generator script adapted to put that in name and vendor. Now that usb_manufacturer and usb_product is only used in pico.json and nanorp2040connect.json where it's actually needed to match what the Arduino-Pico core would use.